### PR TITLE
v0.11.6 -> v1.1.0

### PR DESCRIPTION
**Pull request by GeorgeShagov**
Not tested
I manually copied src archive location from: https://github.com/edenhill/librdkafka/releases

### CODE REVIEW

File: Build.PL [hunk 1]
```diff
@@ -22,8 +22,8 @@ my $builder = Alien::Base::ModuleBuild->new(
     alien_isolate_dynamic => 1,
     alien_repository => {
         protocol => 'https',
-        exact_filename => 'https://github.com/edenhill/librdkafka/archive/v0.11.6.tar.gz',
-        exact_version => 'v0.11.6',
+        exact_filename => 'https://github.com/edenhill/librdkafka/archive/v1.1.0.zip',
```
Comment (plicease) [inline Build.PL]: Is there a reason to use the .zip here instead of .tar.gz?  The latter is usually more reliable than the former.

**Comment by GeorgeShagov** [comment]
No, of cause not. It wasn't an intent. Pls do tar ball.

**Comment by plicease** [comment]
Thanks for updating that.  Can rebase on master?  There is a CI fix that should allow your branch to build cleanly.

**Comment by GeorgeShagov** [comment]
#7

**Comment by plicease** [comment]
btw- in the future you can force-push to the same branch to update the PR, rather than opening a new one.

**Pull request closed without merging**